tests/pipeline/test_13d_filings.py

```diff
@@ -24,9 +24,11 @@
 from zipline.testing.fixtures import WithPipelineEventDataLoader
 from zipline.testing.fixtures import ZiplineTestCase
 
-date_intervals = [[None, '2014-01-04'],
-                  ['2014-01-05', '2014-01-09'],
-                  ['2014-01-10', None]]
+date_intervals = [
+    [['2014-01-01', '2014-01-04'],
```

Comment:
I changed the way `get_values_for_date_ranges` works, so now we need concrete start/end dates. I can infer that logic within `get_values_for_date_ranges` itself (e.g., assume that `None` at the start refers to the start of the month, and `None` at the end refers to the end); I felt it was clearer to just explicitly do this where the date intervals are defined.